src/maxmind.cpp

```diff
@@ -103,6 +140,10 @@ std::vector < std::string > maxmind_bindings::connection(MMDB_s *data, std::vect
   return mmdb_getstring(data, ip_addresses, "connection_type", NULL);
 }
 
+std::vector < int > maxmind_bindings::city_geoname_id(MMDB_s *data, std::vector < std::string >& ip_addresses){
+  return mmdb_getint(data, ip_addresses, "city", "geoname_id", NULL);
```

Comment:
If you're outputting NA_REAL and NA_INTEGER should we not be using IntegerVector and NumericVector types here? Representing NAs in std lib vectors has gone bad sometimes for me.